e2e_test/hawk_test_driver.py

```diff
@@ -533,9 +541,17 @@ def test_click_around_edit_conf(self):
         print("TEST: test_click_around_edit_conf")
         print("TEST: Will click on Constraints, Nodes, Tags, Alerts and Fencing")
         self.check_edit_conf()
-        self.check_and_click_by_xpath("while checking around edit configuration",
-                                      [Xpath.HREF_CONSTRAINTS, Xpath.HREF_NODES, Xpath.HREF_TAGS,
-                                       Xpath.HREF_ALERTS, Xpath.HREF_FENCING])
+
+        click_list = [
+            Xpath.HREF_CONSTRAINTS, Xpath.HREF_NODES, Xpath.HREF_TAGS,
+            Xpath.HREF_ALERTS, Xpath.HREF_FENCING
+        ]
+
+        # HAWK version below 12-SP3 does not provide link to fencing
```

Comment:
Why this? I'm thinking if we should not fix that.

Reply:
If your team plans to fix that we can drop this when it's fixed.

Reply:
sure, we need first to track/understand  the issue then we can fix it.

Reply:
https://github.com/ClusterLabs/hawk/issues/219 can you provide more feedback there?